lib/npm-workspace.js

```diff
@@ -19,7 +19,8 @@ self.cli = function() {
     .option('-c, --copy', 'Copy modules instead of linking')
     .option('-v, --verbose', 'Output verbose log')
     .option('-g, --remove-git', 'Remove .git directories during copy')
-    .option('-p, --production', 'Installs only dependencies (no devDependencies)');
+    .option('-p, --production', 'Installs only dependencies (no devDependencies)')
+    .option('-l, --relative', 'Create relative symlinks for modules instead of absolute ones');
```

Comment:
Used -l to leave -r for other options ( usually -r in cli is for recursive operations, -l is for "local/link" ones )